ceph.spec.in

```diff
@@ -61,6 +78,11 @@ Requires: systemd
 %if 0%{with cephfs_java}
 BuildRequires: sharutils
 %endif
+%if 0%{with selinux}
+BuildRequires:	checkpolicy
+BuildRequires:	selinux-policy-devel
+BuildRequires:	/usr/share/selinux/devel/policyhelp
```

Comment:
- this file belongs to ”selinux-policy-doc“ on my fedora22. but ”selinux-policy-doc“ is not listed in the BuildRequires.
- and this file is not needed anymore to figured out the `_selinux_policy_version` on fedora 20+.